examples/activity-analyzer/src/tests/index.ts

```diff
@@ -23,6 +24,7 @@ describe("Test Example Activity Analyzer", function () {
     };
 
     const itFactory = helpers.itFactory(new MyActivityAnalyzerPlugin(), activityAnalyzerProperties);
+    helpers.mockApi(/\/v1\/activity_analyzers\/(.){1,10}\/properties/).returns(activityAnalyzerProperties);
```

Comment:
Why did you add that?

If it is required for the tests to pass, how could the code without this line land on master previously?

Reply:
Indeed this one is a mistake :) I'll update the PR

Reply:
Did you? =)

Reply:
Sorry about that -> done